prompts/litellm/google-gemini-research.yml

```
@@ -0,0 +1,46 @@
+input_variables:
+  - topic
+  - focus_areas
+  - target_domain
+references:
+  - https://docs.litellm.ai/docs/providers/gemini
+tags:
+  - litellm
+  - google
+  - gemini
+  - research
+  - summary
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)